bluebrain/deployment/environments/applications_viz.yaml

```diff
@@ -13,8 +13,10 @@ spack:
   specs:
     - brayns
     - brayns@3.8.0
+    - brayns@3.8.1
```

Comment:
Is it intentional that this file is one version behind the recipes? And that the previous version stays around? :-)

Reply:
Yeah by default it deploys the latest and remove the old ones.
As Fabien is on holiday, I dunno which versions are still in use so I will keep both 3.8.0 and 3.8.1 and clean it up once he is back.

Reply:
Makes sense, thanks for clarifying!